Library/Formula/gtk+3.rb

```diff
@@ -11,6 +12,8 @@ class Gtkx3 < Formula
   end
 
   option :universal
+  option "with-quartz-relocation", "Build with quartz relocation support"
```

Comment:
I never understood why this option was on by default in the formula. It only makes sense if you intend to put gtk and its dependencies in an app bundle. Otherwise it could actually break a couple of things, especially if HOMEBREW_PREFIX is not /usr/local.